tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -331,6 +331,8 @@
   "DD_TRACE_HTTP_CLIENT_EXCLUDED_URL_SUBSTRINGS": "trace_http_client_excluded_urls",
   "DD_HTTP_SERVER_ERROR_STATUSES": "trace_http_server_error_statuses",
   "DD_HTTP_CLIENT_ERROR_STATUSES": "trace_http_client_error_statuses",
+  "DD_TRACE_HTTP_SERVER_ERROR_STATUSES": "trace_http_server_error_statuses",
+  "DD_TRACE_HTTP_CLIENT_ERROR_STATUSES": "trace_http_client_error_statuses",
```

Comment:
PR opened here? https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json?

Reply:
Yesss waiting for approval along with other configs that need to be consistent:
https://github.com/DataDog/dd-go/pull/151976